### DISCUSSION TITLE
Turborepo Monorepo Setup Example

### DISCUSSION THREAD

**Discussion by kevinswarner** [tailwindlabs/tailwindcss#8332] [Help]
Hi, I am working on getting a basic monorepo set up using Turborepo. (https://turborepo.org/) I am struggling to figure out how to add Tailwind to their basic example. Specifically, I want to add Tailwind support so that...

- You can add Tailwind classes in the UI library
- You can add Tailwind classes to the react code in the Next apps
- They have a shared configuration
- VS code intellisense works for Tailwind in both the UI and Next apps.

I have no problems adding Tailwind to a standalone NextJS app or a standalone UI package, but it would be helpful to have a working example or some documentation on how to best set this up in a Turborepo monorepo. I think part of the issue is there is still not a lot of community q/a on this particular example. Anyway, just a suggestion. If I get an example working, I will come back and post in this discussion.

If anyone has suggestions, that would be great.

Thanks!

**Comment by rvanlaarhoven** [comment]
FWIW, there's [a Tailwindcss example in the turbo repository](https://github.com/vercel/turbo/tree/main/examples/with-tailwind).
